topbeat/docs/fields.asciidoc

```diff
@@ -168,12 +167,12 @@ type: int
 
 The amount of CPU time spent in involuntary wait by the virtual CPU while the hypervisor was servicing another processor. Available only on Unix.
 
-[float]
+
 === cpus Fields
 
 This group contains CPU usage per core statistics.
 
-[float]
```

Comment:
@dedemorton Good to know. I think it is a must that we automate this as otherwise the automatic generation is broken. I'm trying to figure out the logic behind it.

* Is is always added when a new block starts?
* Why do we only have it in topbeat?